modules/orchestrator/con-architecture-overview.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="con-architecture-overview.adoc_{context}"]
+= Architecture overview
+
+The Orchestrator architecture is composed of several integral components, each contributing to the running and management of workflows.
+
+{product} ({product-very-short}):: Serves as the primary interface. {product-custom-resource-type} fulfills the following roles:
+
+* Orchestrator Plugins: Both frontend and backend plugins provide the interface and necessary data retrieval for users to run and monitor workflows within {product-very-short}.
+* Notifications Plugin: Informs users about workflow events.
+
+OpenShift Serverless Logic Operator:: Manages the Sonataflow custom resource (CR), where each CR represents a deployed workflow.
+
+Sonataflow Runtime/Workflow Application:: Functions as a deployed workflow. Operates as an HTTP server, handling requests for running workflow instances. It is managed as a Kubernetes (K8s) deployment by the Openshift Serverless Logic Operator.
```

Comment:
It seems it is a subcomponent managed by the OpenShift Serverless Logic Operator above.

```suggestion
Sonataflow Runtime/Workflow Application::: Functions as a deployed workflow. Operates as an HTTP server, handling requests for running workflow instances. It is managed as a Kubernetes (K8s) deployment by the Openshift Serverless Logic Operator.
```

I am not sure for the next components (Data Index Service, Job Service)